src/components/editable-input/editable-input.jsx

```diff
@@ -136,6 +151,8 @@ class EditableInput extends React.Component {
 EditableInput.propTypes = {
   className: PropTypes.string,
   style: PropTypes.object,
+  /** Needs to be a valid input type */
+  type: PropTypes.string,
```

Comment:
Maybe use:
`type: PropTypes.oneOf(['text', 'tel', 'number', 'email']),`

Reply:
I just picked up the same propTypes validation as was used in the regular input for this prop, will probably leave as is for consistency.